### DISCUSSION TITLE
Επι πτυχίω και φυσική παρουσία

### DISCUSSION THREAD

**Discussion by VaggelisKair** [courses-ionio/hci#1519] [Announcements]
Σε περίπτωση που κάποιος επί πτυχίω δεν μπορεί να βρίσκεται στην Κέρκυρα για αυτό το εξάμηνο πως θα μπορέσει να εξασφαλίσει έναν προβιβάσιμο βαθμό ώστε να περάσει το μάθημα?

**Comment by epidrome** [comment]
σωστή απορία, αλλά μόνο το υπουργείο-ίδρυμα μπορεί να την απαντήσει που έβγαλε τις σχετικές αποφάσεις, χωρίς να σκεφτεί ότι κάποια μαθήματα δεν έχουν τελικές εξετάσεις, το οποίο είναι και το σωστό γενικά.

**Reply by VaggelisKair** [reply]
Άρα με λίγα λόγια δεν θα μπορέσουν αρκετοί φοιτητές να περάσουν το μάθημα αυτό το εξάμηνο εξαιτίας αυτής της αλλαγής, ούτε προσφέρεται κάποιος άλλος τρόπος αξιολόγησης έστω και για χαμηλότερο βαθμό και δεν μπορούμε να κάνουμε και τίποτα γι'αυτό?

Δηλαδή θα πρέπει να περιμένουμε ένα ολόκληρο έτος για να μπορέσουμε να έχουμε ξανά δικαίωμα εξεταστικής και μόνο αν παρευρεθούμε στο μάθημα για ένα ολόκληρο εξάμηνο? 

Όσοι δεν έχουν την οικονομική δυνατότητα?

**Reply by epidrome** [reply]
1. αν κάποιος δεν παρουσιαστεί στην τελική εξέταση μπορεί να περάσει ένα μάθημα;
2. αν η τελική εξέταση δεν είναι σωστός τρόπος διδακτικης πρέπει να την κρατήσουμε;

**Reply by Sola1r3** [reply]
Είμαι και εγώ επί πτυχίω. Δεν βρίσκομαι πλέον Κέρκυρα, δεν μπορώ να βρίσκομαι Κέρκυρα λόγω δουλειάς και από ότι καταλαβαίνω δεν μπορώ να τελειώσω την σχολή επειδή δεν θα έχω φυσική παρουσία σε 2 από τα τελευταία 4 μαθήματα που μου απομένουν??? Για ποιον λόγο ακριβώς γίνεται αυτό; Και τι ικανοποιεί το να γίνονται τα μαθήματα barrier για εμάς που δεν έχουμε πλέον την δυνατότητα να είμαστε εκεί για τον χ y λόγο? Θα μπορούσε να γίνει εξαίρεση πολύ εύκολα, στο χέρι σας είναι κύριε Χωριανόπουλε δεν σας έβαλε κανείς το μαχαίρι στον λαιμό να διεξαχθεί το μάθημα με αυτό τον τρόπο,  θέλω να ασχοληθώ και να περάσω το μάθημα και πρακτικά δεν μπορώ, όταν εσείς ο ίδιος πέρσι μας μιλούσατε για την τηλεργασία και τα θετικά της. Θεωρώ πραγματικά πως δεν υπάρχει κανένας απολύτως λόγος για τον συγκεκριμένο κανονισμό περί φυσικής παρουσίας τουλάχιστον σε ότι αφορά επί πτυχίω φοιτητές. Δώστε μας τουλάχιστον την δυνατότητα να το περάσουμε με 5 χωρίς συμμετοχικό δια ζώσης περιεχόμενο όπως γίνεται δηλαδή τον Σεπτέμβρη.

**Reply by epidrome** [reply]
γιατί δεν ζητάτε από έναν συνάδελφο που έχει τελική εξέταση στο μάθημα του να σας βαθμολογήσει με 5 μόνο για τις εργασίες του εξαμήνου και να μην παρουσιαστείτε στην εξέταση;

το μόνο που άλλαξε (και αυτό προς το καλύτερο) είναι ότι έγιναν υποχρεωτικές οι 2-3 ατομικές παρουσιάσεις, αυτό δίνει αξία στον βαθμό (πτυχίο) σας!

τόσο δύσκολο είναι να έρθετε 2-3 φορές;

**Reply by mariosconsta** [reply]
> γιατί δεν ζητάτε από έναν συνάδελφο που έχει τελική εξέταση στο μάθημα του να σας βαθμολογήσει με 5 μόνο για τις εργασίες του εξαμήνου και να μην παρουσιαστείτε στην εξέταση;
> 
> το μόνο που άλλαξε (και αυτό προς το καλύτερο) είναι ότι έγιναν υποχρεωτικές οι 2-3 ατομικές παρουσιάσεις, αυτό δίνει αξία στον βαθμό (πτυχίο) σας!
> 
> τόσο δύσκολο είναι να έρθετε 2-3 φορές;

Υπάρχουν άτομα σαν και εμένα που δεν είναι Έλληνες και μένουν μόνιμα εξωτερικό. Τα αεροπορικά κοστίζουν + διαμονή στην Κέρκυρα. Και όπως είπε το παιδί είναι και η δουλειά στην μέση που δεν μπορούμε να αφήσουμε. Το πρόβλημα δεν είναι μόνο με το μάθημα σας αλλά με όλα τα μαθήματα. Και εμένα μου έμειναν 4 μαθήματα για πτυχίο. Αυτό είναι το ένα που δεν πέρασα (σας έστειλα σχετικό email).

**Reply by CptMike** [reply]
Κύριε καταλαβαίνετε ότι οι 2-3 φορές, που στην καλύτερη περίπτωση είναι οι παρουσιάσεις να γίνουν συνεχόμενα είναι 3 εβδομάδες; Ξέρετε ότι η μεταφορά και η διαμονή πόσο αξίζει; Ξέρετε πόσες οικογένεις έχουν πληγεί και δε μπορούν πλέον να υποστηρίζουν τα παιδιά τους στην Κέρκυρα, η οποία είναι και πανάκριβη; 

Στην τελική δε ζητάμε να μας περάσετε το μάθημα. Δικαίωμα να περάσουμε δίνοντας τις εργασίες όπως όλους τους φοιτητές θέλουμε. Ό,τι και να λέει το υπουργείο, λίγη κατανόηση ζητάμε.

**Reply by epidrome** [reply]
αυτό που καταλαβαίνω είναι ότι ζητάτε από ένα ίδρυμα που (από τον νόμο του υπουργείου) δεν παρέχει εξ αποστάσεως εκπαίδευση να περάσετε χωρίς παρουσία;

προσωπικά προτιμώ το εξ αποστάσεως γιατί ταιριάζει σε αυτό το μάθημα, αν το υπουργείο-ίδρυμα μας δώσει αυτήν την δυνατότητα ευχαρίστως!

**Reply by CptMike** [reply]
> αυτό που καταλαβαίνω είναι ότι ζητάτε από ένα ίδρυμα που (από τον νόμο του υπουργείου) δεν παρέχει εξ αποστάσεως εκπαίδευση να περάσετε χωρίς παρουσία;
> 
> προσωπικά προτιμώ το εξ αποστάσεως γιατί ταιριάζει σε αυτό το μάθημα, αν το υπουργείο-ίδρυμα μας δώσει αυτήν την δυνατότητα ευχαρίστως!

Αν δε κάνω λάθος, το υπουργείο απαγόρευσε την τηλεκπαίδευση και το υδριβικό σύστημα. Δεν είναι εντολή του υπουργείου να καθίσταται η φυσική παρουσία, ως απαραίτητο κριτήριο για την επιτυχή ολοκλήρωση του μαθήματος.

**Reply by p15bako** [reply]
Απ ότι καταλαβαίνω, το κομμάτι της σύγχρονης συμμετοχής στην τάξη και ατομικές παρουσιάσεις προόδου χωρίζεται σε 2 κομμάτια. Το πρώτο που έγινε με classtime το περασμένο εξάμηνο και δεύτερον, οι παρουσιάσεις. Θα μπορούσε τουλάχιστον το classtime να γίνει και εξ αποστάσεως για τους επι πτυχίω; Γιατί ακόμα και να έρθει κάποιος στις παρουσιάσεις και δεν έχει συμμετάσχει στο classtime, το 30% γίνεται αυτόματα 15% θεωρώ. Και με βάση το "Θα πρέπει να έχετε συμπληρώσει την βάση σε κάθε ένα από τα παραπάνω κριτήρια.", αν η μία παρουσίαση πάρει βαθμό λιγότερο του άριστου (δηλαδή 10), είμαστε ήδη κάτω απ την βάση, οπότε είναι πρακτικά αδύνατο να περάσουμε το μάθημα.

**Reply by Sola1r3** [reply]
> 
> 
> γιατί δεν ζητάτε από έναν συνάδελφο που έχει τελική εξέταση στο μάθημα του να σας βαθμολογήσει με 5 μόνο για τις εργασίες του εξαμήνου και να μην παρουσιαστείτε στην εξέταση;
> 
> το μόνο που άλλαξε (και αυτό προς το καλύτερο) είναι ότι έγιναν υποχρεωτικές οι 2-3 ατομικές παρουσιάσεις, αυτό δίνει αξία στον βαθμό (πτυχίο) σας!
> 
> τόσο δύσκολο είναι να έρθετε 2-3 φορές;

Να παρουσιάσω την δουλειά μου ναι κανένα πρόβλημα. Αυτό μπορεί να επιτευχθεί και με πλατφόρμες τύπου ζουμ για παράδειγμα. Τώρα όσον αφορά αποφάσεις του υπουργείου και του ιδρύματος και τον περιορισμό που υπάρχει γύρω σας στο να μας παρέχεται και μας την ευκαιρία να συμμετέχουμε στο μάθημα, θέλετε με αρνητικό αντίκτυπο στον βαθμό, ε έστω έτσι, δεν υπάρχει ούτε ένα άλλο μάθημα που να απαιτεί φυσική παρουσία από επί πτυχίω φοιτητές. Άρα δεν βλέπω πουθενά κανέναν να σας απαγορεύει να μας επιτρέψετε την συμμετοχή σε τελική εξέταση του μαθήματος. 

Ζητάω την κατανόηση σας.

**Reply by epidrome** [reply]
η κατανόηση βρίσκεται ήδη στο μέγιστο, αλλά δεν είναι αυτό που πράγματι ζητάτε, το οποίο είναι να προσαρμοστεί ένα μάθημα στις δικές σας προτιμήσεις
